autodocodec-api-usage/test_resources/json-schema/expression.json

```diff
@@ -0,0 +1,66 @@
+{
+    "$defs": {
+        "Expression": {
+            "$comment": "Expression",
+            "oneOf": [
```

Comment:
Does `json-schema` not have discriminated unions?

Reply:
I don't think it does. I think it's an OpenAPI3 extension

Reply:
You're right: https://stackoverflow.com/questions/60959556/using-discriminator-keyword-in-json-schema